inst/css/BS4/pkgdown-variables.scss

```diff
@@ -0,0 +1,26 @@
+// This Sass file wants to be compiled with Bootstrap Sass headers
+// All custom properties are prefixed with `rs-` (abbr: Rmd style) to avoid conflicts
```

Comment:
@maelle what do you think about using `pkgdown-` instead of `rs-`?

Reply:
That could work but would it prevent easier re-use by other formats @cderv?

Reply:
We don't really know yet what are the generic variable names we want to provide generic theming system across outputs. It is possible that we don't reuse those name here anyway. If we do we'll be able to provide an upper layer of variables I guess. 

So I don't have a strong opinion on this. For now, this is quite specific to pkgdown and a `pkgdown-` prefix would make it clear that this is for pkgdown theming. `rs-` is not so clear from a user point of view IMO

Reply:
I second the `pkgdown-` prefix for now :)